apps/get-files/src/get_files.py

```diff
@@ -33,7 +33,7 @@ def get_products(files):
         'size': item['size'],
         'filename': item['filename'],
         's3': item['s3'],
-    } for item in files if item['file_type'] == 'product']
+    } for item in files if item['file_type'] == 'product' and splitext(item['filename'])[-1] in ('.zip', '.nc')]
```

Comment:
maybe `and Path(item['filename']).suffix in ('.zip', '.nc')`?

Reply:
or even:
```
def is_product(filename, file_type):
    return file_type == 'product' and Path(filename) in ('.zip', '.nc)
 
.... for item in files if is_product(item['filename'], item['file_type'])

Reply:
I originally did it with splitext because other parts of this code are `os.path` based -- `pathlib` isn't being used. 

I also left if a little dirty to intentionally be obnoxious as we shouldn't persist this change. 

I've changed it to a `visible_product` helper function with `pathlib`. I've settled on rather having a cleaner "dirty" implementation in case it actually persists much longer than it should.